tests/two_sum_test.py

```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.two_sum import twoSum
+
```

Comment:
Great start! I suggest thinking about possible edge cases and writing tests for those as well!